tools/releaseBuild/Images/microsoft_powershell_windowsservercore/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN Import-Module ./containerFiles/dockerInstall.psm1; `
 
 # Install WIX
 ADD https://github.com/wixtoolset/wix3/releases/download/wix311rtm/wix311-binaries.zip /wix.zip
+ADD https://pscoretestdata.blob.core.windows.net/build-files/makeappx/makeappx.tgz?sp=r&st=2019-04-03T20:37:29Z&se=2020-04-04T04:37:29Z&spr=https&sv=2018-03-28&sig=wGY3gCjVcyowIfGVKWSdiOlkXMrG243YBp7LWrMB2BY%3D&sr=b /makeappx
```

Comment:
change this to a zip and expand it with expand-archive

Reply:
Does renaming to .tar.gzip make it just work or does `ADD` not do what the documentation says?

Reply:
I didn't try renaming to tar.gz.  It may just not work on windows

Reply:
After discussion, changing to .zip since it appears that auto-expanding of tgz doesn't work in dockerfile